packages/venia-ui/lib/components/MegaMenu/submenu.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { mergeClasses } from '../../classify';
+import defaultClasses from './submenu.css';
+import SubmenuColumn from './submenuColumn';
+import PropTypes from 'prop-types';
+
+/**
+ * The Submenu component displays submenu in mega menu
+ *
+ * @param {array} props.items - categories to display
+ * @param {int} props.mainNavWidth - width of the main nav. It's used for setting min-width of the submenu
+ */
+const Submenu = props => {
+    const { items, mainNavWidth } = props;
+    const PADDING_OFFSET = 20;
```

Comment:
Is this a constant that one can configure either in code or through the admin panel?

Reply:
This can't be configured in the Admin panel.